From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Sat, 29 Oct 2011 18:53:38 +0200
Subject: [PATCH] Fix crosscompilation

This patch fixes cross compilation, of another handcrafted
Makefile.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
#20081125: wsa: Mailed to upstream.
#20081127: wsa: Accepted upstream. Wait for next release.
#20100317: mkl: Mailed to upstream again.

 Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 669cbcd..f4c4868 100644
--- a/Makefile
+++ b/Makefile
@@ -15,8 +15,8 @@ CRNPATH = /etc/cron.d
 ROTPATH = /etc/logrotate.d
 
 # CFLAGS  = -g -I. -Wall	 # -DHTTPSTATS
-CFLAGS  = -O -I. -Wall	 # -DHTTPSTATS
-LDFLAGS = -lncurses -lm -lz
+CFLAGS  += -O2 -I. -Wall	 # -DHTTPSTATS
+LDFLAGS += -lncurses -lm -lz
 OBJMOD0 = version.o
 OBJMOD1 = various.o  deviate.o   procdbase.o
 OBJMOD2 = acctproc.o photoproc.o photosyst.o  rawlog.o ifprop.o parseable.o
@@ -27,7 +27,7 @@ ALLMODS = $(OBJMOD0) $(OBJMOD1) $(OBJMOD2) $(OBJMOD3) $(OBJMOD4)
 all: 		atop
 
 atop:		atop.o    $(ALLMODS) Makefile
-		cc atop.o $(ALLMODS) -o atop $(LDFLAGS)
+		$(CC) atop.o $(ALLMODS) -o atop $(LDFLAGS)
 
 clean:
 		rm -f *.o
